packages/contracts-bedrock/src/L1/OptimismPortal.sol

```diff
@@ -45,7 +45,7 @@ contract OptimismPortal is Initializable, ResourceMetering, ISemver {
     uint64 internal constant RECEIVE_DEFAULT_GAS_LIMIT = 100_000;
 
     /// @notice The L2 gas limit for system deposit transactions that are initiated from L1.
-    uint32 internal constant SYSTEM_DEPOSIT_GAS_LIMIT = 80_000;
+    uint32 internal constant SYSTEM_DEPOSIT_GAS_LIMIT = 200_000;
```

Comment:
Do we have tests that make sure this value remains high enough? How did we miss this and how did we catch it?

Reply:
I don't think there is a good way to catch this in solidity. I knew that this was a possibility and the set of e2e tests defined in https://github.com/ethereum-optimism/optimism/pull/10310#discussion_r1589713371 where crafted to catch such an issue.

It could be possible to do in solidity with `--isolate` mode where you check how much gas is used and assert its less than the constant.